CHANGES.txt

```diff
@@ -35,6 +35,8 @@ Trunk
     FALCON-1770 Update README file (Ajay Yadava)
 
   BUG FIXES
+    FALCON-1842 Falcon build failed in Jenkins at OozieFeedWorkflowBuilderTest (Balu Vellanki)
```

Comment:
Sometimes we are adding and sometimes not. @pallavi-rao  @sandeepSamudrala  What is the final call on this